spec/spec.md

```diff
@@ -123,6 +123,55 @@ Database |
 Finalize the component stack list - are these correct? Are we missing any?
 :::
 
+## Addressing
+
+A user's logical Identity Hub (their collection of Hub Instances) can be addressed in many ways, 
+but the mechanisms below ****MUST**** be supported by a compliant Identity Hub implementation:
+
+### DID-Relative URLs
+
+The following DID URL constructions are used to address [[ref: Hub Instances]] found to be associated 
+with a given DID, as located via the DID resolution process.
+
+*DID-Relative URL addressing a single file:*
+
+```json
+did:example:123?service=IdentityHub&resource=Qm3fw45v46w45vw54wgwv78jbdse4w
+```
+
+*DID-Relative URL addressing of a Collection of the same type:*
+
+```json
+did:example:123?service=IdentityHub&collection=https://schema.org/MusicPlaylist
+```
+
+*DID-Relative URL addressing multiple Collections of different types:*
+
+```json
+did:example:123?service=IdentityHub&collection=https://schema.org/MusicPlaylist&collection=https://schema.org/Event
+```
+
+*DID-Relative URL addressing Collections and specific resources together:*
+
+```json
+did:example:123?service=IdentityHub&collection=https://schema.org/MusicPlaylist&resource=Qm3fw45v46w45vw54wgwv78jbdse4w
+```
+
+::: note
+For example purposes, the parameters above are not URL encoded, but should be when using Identity Hub URLs in practice.
```

Comment:
```suggestion
For example purposes, the parameters above are not URL encoded, but should be when using Identity Hub URLs in practice.
```
I suggest putting this note *before* the content to which it applies.  I also suggest that the example URLs be shown exactly as they are intended to appear in use, and that the unencoded parameter(s) be shown alongside the relevant example.